.gitignore

```diff
@@ -1,4 +1,6 @@
 .psci
 .psci_modules
+.history
```

Comment:
I think this should go into your user specific git ignore (which can then be global and not added to every project)